mmdet3d/core/bbox/structures/coord_3d_mode.py

```diff
@@ -220,9 +220,11 @@ def convert_point(point, src, dst, rt_mat=None):
         # convert point from `src` mode to `dst` mode.
         # TODO: LIDAR
         # only implemented provided Rt matrix in cam-depth conversion
-        if src == Coord3DMode.LIDAR and dst == Coord3DMode.CAM:
+        if (src == Coord3DMode.LIDAR
+                and dst == Coord3DMode.CAM) and rt_mat is None:
             rt_mat = arr.new_tensor([[0, -1, 0], [0, 0, -1], [1, 0, 0]])
-        elif src == Coord3DMode.CAM and dst == Coord3DMode.LIDAR:
+        elif (src == Coord3DMode.CAM
+              and dst == Coord3DMode.LIDAR) and rt_mat is None:
             rt_mat = arr.new_tensor([[0, 0, 1], [-1, 0, 0], [0, -1, 0]])
         elif src == Coord3DMode.DEPTH and dst == Coord3DMode.CAM:
             if rt_mat is None:
```

Comment:
Also, maybe it's helpful to add a comment here to point out why LIDAR-CAM conversion is different from DEPTH-CAM conversion? You can say that SUNRGB-D camera calibration files are different from that of KITTI, and currently we keep this hack.

Reply:
Thanks for your advices. I have updated the place of `rt_mat is None` and added the comment to clarify the difference between LIDAR-CAM conversion and DEPTH-CAM conversion.